tests/test_core/conftest.py

```diff
@@ -75,15 +16,66 @@ async def p9(request, tmpdir):
     await ctx.close()
 
 
+@pytest.fixture
+def nsname():
+    ns = uifname()
+    netns.create(ns)
+    with IPRoute(netns=ns) as ipr:
+        ipr.link('set', index=1, state='up')
+        ipr.poll(ipr.addr, 'dump', address='127.0.0.1', timeout=5)
+    yield ns
+    try:
+        netns.delete(ns)
+    except Exception:
+        pass
+
+
+@pytest.fixture
+def link(nsname):
```

Comment:
Type annotations for test fixtures, even if they're not checked (yet !), really help me understand what they do at a glance even when there's no doc.

Reply:
oh, you urge me to use annotations

ok, I'll try to change my habits :)